Void doesn't seem to ship rustfmt, so disable its bindings.

--- a/build.rs
+++ b/build.rs
@@ -9,7 +9,7 @@
 
     let python_cfg = PythonConfig::new("etebase_python".to_owned());
     let swig_gen = flapigen::Generator::new(LanguageConfig::PythonConfig(python_cfg))
-        .rustfmt_bindings(true);
+        .rustfmt_bindings(false);
     swig_gen.expand("python bindings", &in_src, &out_src);
     println!("cargo:rerun-if-changed={}", in_src.display());
 }
